### PR TITLE
Merge master to 8.0.0

### PR DESCRIPTION
**Pull request by agliga**
## Description

Did a merge form master to pull in unchecked icon color.

Changed icon in the new icon variable file (in ds6). 
Re-removed all those old variable files (were added again from merge)
Changed color variable to match new syntax.

### CODE REVIEW

File: src/less/variables/ds6/icon-variables.less [hunk 1]
```diff
@@ -254,7 +254,7 @@
...
 
-@icon-checkbox-unchecked-base64-light: 'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiANCiAgd2lkdGg9IjE1Ljc3IiBoZWlnaHQ9IjE1LjgiIHZpZXdCb3g9IjQuMTIgNC4xIDE1Ljc3IDE1LjgiPjxwYXRoIGZpbGw9IiNmZmYiIGQ9Ik0xOC41NiA1LjI1SDUuNDRhLjE5LjE5IDAgMCAwLS4xOS4xODd2MTMuMTI2YS4xOS4xOSAwIDAgMCAuMTkuMTg3aDEzLjEyYS4xOS4xOSAwIDAgMCAuMTktLjE4N1Y1LjQzN2EuMTkuMTkgMCAwIDAtLjE5LS4xODd6TTQuMTE2IDE5LjloMTUuNzY4VjQuMUg0LjExNnYxNS44eiIvPjwvc3ZnPg==';
+@icon-checkbox-unchecked-base64-light: 'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iMTUuNzciIGhlaWdodD0iMTUuOCIgdmlld0JveD0iNC4xMiA0LjEgMTUuNzcgMTUuOCI+PHBhdGggZmlsbD0iI2M4YzhjOCIgZmlsbC1ydWxlPSJldmVub2RkIiBkPSJNMTguNTYgNS4yNUg1LjQ0YS4xOS4xOSAwIDAgMC0uMTkuMTg3djEzLjEyNmEuMTkuMTkgMCAwIDAgLjE5LjE4N2gxMy4xMmEuMTkuMTkgMCAwIDAgLjE5LS4xODdWNS40MzdhLjE5LjE5IDAgMCAwLS4xOS0uMTg3ek00LjExNiAxOS45aDE1Ljc2OFY0LjFINC4xMTZ2MTUuOHoiLz48L3N2Zz4=';
```
Comment (seangates) [inline src/less/variables/ds6/icon-variables.less]: This shouldn't change.

File: src/svg/ds6/icons.svg [hunk 2]
```diff
@@ -31,7 +31,7 @@
         <path d="M12.186 18.285c-.451-.009-.809-.167-1.075-.441l-9.337-9.6a1.527 1.527 0 0 1-.424-.999v-.108c.015-.386.166-.741.424-1.008.56-.573 1.529-.57 2.082 0l8.294 8.53 8.292-8.532c.558-.57 1.526-.57 2.08 0 .265.27.416.629.428 1.01v.087c-.012.391-.165.75-.427 1.02l-9.333 9.6a1.443 1.443 0 0 1-1.004.441"/>
     </symbol>
     <symbol id="icon-checkbox-unchecked" viewBox="4.12 4.1 15.77 15.8">
-        <path fill-rule="evenodd" d="M18.56 5.25H5.44a.19.19 0 0 0-.19.187v13.126a.19.19 0 0 0 .19.187h13.12a.19.19 0 0 0 .19-.187V5.437a.19.19 0 0 0-.19-.187zM4.116 19.9h15.768V4.1H4.116v15.8z"/>
+        <path fill="#c8c8c8" fill-rule="evenodd" d="M18.56 5.25H5.44a.19.19 0 0 0-.19.187v13.126a.19.19 0 0 0 .19.187h13.12a.19.19 0 0 0 .19-.187V5.437a.19.19 0 0 0-.19-.187zM4.116 19.9h15.768V4.1H4.116v15.8z"/>
```
Comment (seangates) [inline src/svg/ds6/icons.svg]: This shouldn't change.

Comment (seangates) [reply src/svg/ds6/icons.svg]: It appears this is my fault. Let me fix it.

Comment (seangates) [reply src/svg/ds6/icons.svg]: Fixed here: https://github.com/eBay/skin/pull/740/files#diff-0414bda8012ba8a5234866832070991aR34

Comment (agliga) [reply src/svg/ds6/icons.svg]: Pushed those changes @seangates, let me know if it's fine now.
Not sure about the windows high contrast one though.

**Comment by agliga** [comment]
@seangates I just updated again from master, can you check agqin in case I missed something? I  deleted the dist variable files, so in case something is missing let me know so I can add it back.

**Comment by seangates** [comment]
LGTM.

**Pull request merged**